static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.ts

```diff
@@ -44,6 +45,7 @@ initMessenger(
 		passback,
 	],
 	[scroll, viewport],
+	reportError,
```

Comment:
Is this new?

Reply:
Yes, I mentioned in the commercial-core PR:

> I thought passing in the raven/sentry error handler would be better rather than copying it over, but not totally sure.
> _Originally posted by @Jakeii in https://github.com/guardian/commercial-core/pull/677#discussion_r977417736_